client/src/locales/fr.ts

```diff
@@ -0,0 +1,329 @@
+import { OttWebsocketError } from "ott-common/models/types";
+
+export default {
+	"common": {
+		"yes": "Oui",
+		"no": "Non",
+		"cancel": "Annuler",
+		"close": "Fermer",
+		"close-all": "Tout Fermer",
+		"add": "Ajouter",
+		"remove": "Retirer",
+		"delete": "Supprimer",
+		"save": "Enregistrer",
+		"search": "Rechercher",
+		"undo": "Retour",
+	},
+	"landing": {
+		hero: {
+			title: "S'amuser ensemble !",
+			description:
+				"Vidéo synchronisée en temps réel. Système de vote optionnel.Thème Sombre. Pas de connexion requise. Tout est OpenSource. Il n'as jamais été aussi simple de regarder des vidéos ensemble!",
+			btns: {
+				create: "@:nav.create.title",
+				browse: "Parcourir les salles",
+				source: "Voir les sources",
+			},
+		},
+		intro: {
+			title: "Regarder des vidéos ensemble à distance est devenue beaucoup plus facile.",
+			name: "OpenTogetherTube",
+			text1: "Plateforme de synchronisation de vidéo en temps réel. Facile d'utilisation et sans connexion. Créer simplement une salle, ajoute une vidéo et invite t'es amis. BOOM! Tu es prêt pour regarder des vidéos avec t'es potes jusqu'à 3h du mat.",
+			text2: "Le TogetherTube original est aimé pour son interface très simpliste, et sa rapidité pour lancer des vidéos. OpenTogetherTube vise à être tout aussi simple et à s'améliorer pour qu'il devienne encore meilleur au fil du temps.",
+			text3: "Actuellement, vous pouvez regarder des vidéos depuis : Youtube, Vimeo, Dailymotion, liens directs vers des vidéos .mp4, ...",
+			link: "d'autres sources arrivent bientôt",
+		},
+		features: {
+			"title": "Fonctionnalités Principales",
+			"syncronized-playback": {
+				title: "Vidéo synchronisée",
+				text: "Appuies sur le bouton play et la vidéo se lance pour tout le monde dans la salle. Aussi simple que ça.",
+			},
+			"permanent-rooms": {
+				title: "Salles Permanentes",
+				text: "Toi et t'es amis êtes des utilisateurs réguliers ? Tu peux créer une salle permanente qui te permettra de conserver la même URL pour ta salle",
```

Comment:
Is it normal to have a space before the "?" in french? Seems inconsistent with other punctuation.

Reply:
Effectively we add a space before a ?  😁 French uses a lot of characters with accents if that's a problem I can remove them but it can make the text less easy to read

Reply:
Ah, I see. We'll leave it as is then.